packages/turf-points-within-polygon/index.js

```diff
@@ -1,14 +1,14 @@
 import pointInPolygon from "@turf/boolean-point-in-polygon";
-import { featureCollection } from "@turf/helpers";
-import { geomEach, featureEach } from "@turf/meta";
+import { featureCollection, multiPoint } from "@turf/helpers";
+import { geomEach, featureEach, coordEach } from "@turf/meta";
 
 /**
- * Finds {@link Points} that fall within {@link (Multi)Polygon(s)}.
+ * Finds {@link Points} or {@link MultiPoint} coordinate positions that fall within {@link (Multi)Polygon(s)}.
  *
  * @name pointsWithinPolygon
- * @param {Feature|FeatureCollection<Point>} points Points as input search
- * @param {FeatureCollection|Geometry|Feature<Polygon|MultiPolygon>} polygons Points must be within these (Multi)Polygon(s)
- * @returns {FeatureCollection<Point>} points that land within at least one polygon
+ * @param {Feature|FeatureCollection<Point|MultiPoint>} points (Multi)Point(s) as input search
```

Comment:
The typescript version of this would be `Feature<Point|MultiPoint> | FeatureCollection<Point|Multipoint>` but maybe this is fine for JSDoc?

Reply:
I agree, as a TS user myself, your suggestion is more correct but looking at the docs for different functions I sensed there was an established convention already.  I don't mind changing them, I would just change the params too to match.  Since each type becomes a link in the docs, maybe it was done for simplicity.

![image](https://user-images.githubusercontent.com/81816/124619663-b1093f80-de2d-11eb-82f3-c34e010b3edf.png)